patch/MultiDelay/MultiDelay.cpp

```diff
@@ -0,0 +1,129 @@
+#include "daisysp.h"
+#include "daisy_patch.h"
+#include <string>
+
+#define MAX_DELAY static_cast<size_t>(48000 * 1.f)
+
+using namespace daisy;
+using namespace daisysp;
+
+DaisyPatch patch;
+
+DelayLine <float, MAX_DELAY>  DSY_SDRAM_BSS delay[3];
+
+struct delTime{
+    float currentDelay;
+    float delayTarget;
+};
+
+delTime delTimes[3];
+Parameter params[4];
```

Comment:
Similar to what you did with the QuadEnvelopes, you could make a struct for all of this. (You could even add Init/Read/Write methods to avoid having to do loops of three in a bunch of different spots).

You can then stick the entire struct in external memory or use DelayLine pointers if you want only the delays in external ram.